app/models/message.rb

```diff
@@ -3,7 +3,8 @@ class Message < ApplicationRecord
   belongs_to :trip
 
   validates_presence_of :trip, :sender_name, :sender_email, :body
-
+  validates_format_of :sender_email, with: /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
```

Comment:
Hmmm.. I'm not for the regex storage as we could want to futher check the email adress (blacklist or else) the creation of a custom validator seems nice but is something I have to learn.
any clues ? where do I put this validating function ?